webui/app/templates/index.html

```diff
@@ -120,15 +120,10 @@ <h2 class="text-center">Conversion Tool Web UI</h2>
         </div>
     {% endif %}
 {% endwith %}
+<script src="{{ url_for('static', filename='js/libs/jquery.min.js') }}" type="text/javascript"></script>
+<script src="{{ url_for('static', filename='js/libs/fileinput.min.js') }}" type="text/javascript"></script>
+<script src="{{ url_for('static', filename='js/libs/bootstrap.min.js') }}" type="text/javascript"></script>
+<script src="{{ url_for('static', filename='js/libs/notify.min.js') }}" type="text/javascript"></script>
+<script src="{{ url_for('static', filename='js/index.js') }}" type="text/javascript"></script>
 </body>
-<script src="{{ url_for('static', filename='js/libs/jquery.min.js') }}"
-        type="text/javascript"></script>
-<script src="{{ url_for('static', filename='js/libs/fileinput.min.js') }}"
-        type="text/javascript"></script>
-<script src="{{ url_for('static', filename='js/libs/bootstrap.min.js') }}"
-        type="text/javascript"></script>
-<script src="{{ url_for('static', filename='js/libs/notify.min.js') }}"
-        type="text/javascript"></script>
-<script src="{{ url_for('static', filename='js/index.js') }}"
-        type="text/javascript"></script>
-</html>
```

Comment:
For this one, where you have multiple small commits doing the same thing, James introduced me to `git commit --amend`, which will add your latest updates to the most recent commit and allow you to use the same message / write a new message.

For more complicated squashing of commits, we use `git rebase -i HEAD~#` (replacing "#" with 5 will allow you to combine the last 5 commits, and replacing both "HEAD" and "#" should let you choose any commit range).

![screen shot 2017-12-12 at 11 27 14 am](https://user-images.githubusercontent.com/13045020/33896025-ceda4186-df2f-11e7-9c9b-e156394abead.png)